### DISCUSSION TITLE
Allow setting the schedule block data via the schedule block dialog

### DISCUSSION THREAD

**Discussion by EuleMitKeule** [home-assistant/frontend#22793] [Configuration] [answered]
In core 2024.10 a feature was added that allows users to define arbitrary data for a schedule block in a schedule entity. This data should also be able to be shown or manipulated via the schedule block form/dialog.

**Comment by karwosts** [comment, marked as answer]
Implemented in https://github.com/home-assistant/frontend/pull/22798

**Reply by EuleMitKeule** [reply]
Wow, that's great! Thank you.
